README.md

```diff
@@ -313,11 +313,13 @@ import {ValidateNested} from "class-validator";
 export class Post {
 
     @ValidateNested()
-    user: User;
+    user: User = new User();
```

Comment:
Again misleading code. I think note below this section is enough.

Reply:
Removed.